hacking/test-module.py

```diff
@@ -196,11 +192,10 @@ def ansiballz_setup(modfile, modname, interpreters):
         sys.exit(err)
     debug_dir = lines[1].strip()
 
-    # All the directories in an AnsiBallZ that modules can live
     core_dirs = glob.glob(os.path.join(debug_dir, 'ansible/modules'))
     collection_dirs = glob.glob(os.path.join(debug_dir, 'ansible_collections/*/*/plugins/modules'))
 
-    # There's only one module in an AnsiBallZ payload so look for the first module and then exit
+    # Find and return the module
```

Comment:
Code comments should not repeat what code does, but rather explain motivation behind the decisions. Please, don't remove such information..